### ISSUE TITLE
New PBT Device Request - <IR blaster>

### ISSUE THREAD

**Issue by twanjaarsveld** [MarkWattTech/Tuya-v2-Supported-Devices#26]
<table class="table">
<tr>
      <td>Remote</td>
      <td>China</td>
      <td>IR blaster</td>
      <td>IR Remote</td>
      <td>?</td>
      <td>https://www.aliexpress.com/item/4000139595706.html?spm=a2g0s.9042311.0.0.27424c4dtXhy6u</td>
      <td>Smart IR blaster</td>
      <td>A link to an image of the product</td>
    </tr>
</table>

**Comment by MarkWattTech** [comment]
is this working in v2? :)

**Comment by twanjaarsveld** [comment]
> is this working in v2? :)

Yes it is, you have to make the scenes in the tuya/smart life app, but you can trigger them using home assistant, im using it to control my tv and stereo system and it works fast and everytime

**Comment by MarkWattTech** [comment]
Yeah I thought it would be done through scenes. I will add this to the
description so that people
Know.

**Comment by twanjaarsveld** [comment]
> Yeah I thought it would be done through scenes. I will add this to the description so that people Know.

Hope they add better support and input using Services in Home assistant soon, but until then it works decently
